solr_helpers/__init__.py

```diff
@@ -403,7 +405,7 @@ def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_
             mutation_filters[attr] = values
         else:
             main_filters[attr] = values
-
+    #ISB-CGC legacy??? Is this applicable in IDC??? Might lead to problems
     # Mutation filters
```

Comment:
Yes, these are going to be implemented. What problems are you anticipating?

Reply:
Right now we do not have any 'mutation' filters so this block is not being used. If we had an attribute that happened to end with 'MUT' could that mess things up